pandas/tests/io/excel/test_odf.py

```diff
@@ -36,3 +36,11 @@ def test_read_writer_table():
     result = pd.read_excel("writertable.odt", "Table1", index_col=0)
 
     tm.assert_frame_equal(result, expected)
+
+
+def test_nonexistent_sheetname_raises(self, read_ext):
```

Comment:
The use of `self` here is causing the error. This isn't a class method so that's what is failing CI.

If you remove I think should turn green

Reply:
should be done in c45ffb3